WordPressAuthenticator/Signin/LoginViewController.swift

```diff
@@ -216,6 +216,10 @@ open class LoginViewController: NUXViewController, LoginFacadeDelegate {
             return
         }
 
+        // Make sure we don't provide any old nonce information when we are required to present only the multi-factor code option.
+        loginFields.nonceInfo = nil
+        loginFields.nonceUserID = 0
```

Comment:
(Nit) It doesn't seem like the optimal place to clear the state. It should probably be cleared right after you are returned to the login screen after a failed attempt.